source/includes/_feature-deriving.md

```diff
@@ -395,4 +395,158 @@ POST'ing to the private variables catalog a Shoji Entity containing a ZCL functi
 
 ### Other transformations
 
```

Comment:
I did the functions that return columns, those are functions that do not return columns but masks. We had a ticket to make columns out of boolean ops but we don't have them yet. They cannot be used to create new variables.